frontend/src/libs/components/radio/radio.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  type Control,
+  type FieldErrors,
+  type FieldPath,
+  type FieldValues,
+} from 'react-hook-form';
+
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useFormController } from '~/libs/hooks/hooks.js';
+
+import common from './common.module.scss';
+import radio from './radio.module.scss';
+
+const styles = { ...common, ...radio };
+
+type Properties<T extends FieldValues, N extends FieldPath<T>> = {
+  control: Control<T, null>;
+  errors?: FieldErrors<T>;
+  label?: string;
+  name: N;
+  beDisabled?: boolean;
```

Comment:
```suggestion
  isDisabled?: boolean;
```
it's one of the`be` verb forms

Reply:
It was one of your previous requests. I wanted them to be "isDisabled" from the very beginning.